pubspec.yaml

```diff
@@ -10,7 +10,7 @@ dependencies:
     git: https://github.com/dart-atom/atom.dart.git
   path: ^1.3.0
   pub_semver: ^1.2.1
-  serial_port: ^0.3.1
+  #serial_port: ^0.3.1 << bundled
```

Comment:
I see travis is unhappy: https://travis-ci.org/dartino/atom-dartino/builds/110855787#L244. Perhaps move this to a dev_dependency? Or keeping it as a regular dep. It shouldn't make it's way into the compiled dart2js artifact.